docs/en/observability/apm/view-and-analyze/ui-overview/services.asciidoc

```diff
@@ -31,12 +31,11 @@ Service groups are {kib} space-specific and available for any users with appropr
 [role="screenshot"]
 image::./images/apm-service-group.png[Example view of service group in the Applications UI in Kibana]
 
-To enable Service groups, open {kib} and navigate to **Stack Management** → **Advanced Settings** → **Observability**,
-and enable the **Service groups feature**.
+//QUESTION FOR COLLEEN or BRANDON: Can you confirm that service groups are enabled by default now? (I don't see the setting for enable service groups under advanced settings, so I have removed the section that describes how to do that.)
```

Comment:
I'll look into this and report back shortly.

Reply:
service groups are enabled in 8.16 by default. Still beta, but they appear without needing to go to advanced settings.
cc: @colleenmcginnis and @dedemorton

Reply:
Thanks @eedugon!